controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -167,6 +172,10 @@ module.exports = async (req, res) => {
             ...userLocation,
             viewtype: 'user'
           });
+
+          if (!allUserIds.includes(userLocation.user_id)) {
+            allUserIds.push(userLocation.user_id);
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issue as above regarding the use of `Array.prototype.includes()`. Please consider using a `Set` for `allUserIds`.
```diff
- if (!allUserIds.includes(userLocation.user_id)) {
-   allUserIds.push(userLocation.user_id);
- }
+ allUserIds.add(userLocation.user_id);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->